cmd/sync.js

```diff
@@ -136,9 +140,15 @@ const handler = function(argv) {
         process.exit(1)
       }
 
+      // converting comma-separated list of labels to array
+      const labels = argv.labels ? argv.labels.split(',') : ['trira']
+      
       return trelloCards.get(configuration.trello, argv.boardRegexp, argv.listRegexp, argv.cardRegexp)
         .then(cards => {
           logger.debug(`${argv.dryRun ? 'Would' : 'Will'} create ${cards.length} issues in JIRA`)
+
+          cards.forEach(card => {card.labels = card.labels.concat(labels)})
```

Comment:
@trepel this was problem with previous implementation as well - but I can see that while labels are unique, here we are able to more easily define one label twice - I'm not use if JIRA won't choke on that

Reply:
@kpiwko My understanding of your comment is that the code does not handle the situation when the labels array contain duplicates. Thanks for pointing this out, I haven't thought about it to be honest.

I tried and JIRA handles such situation well - no errors, only one label (regardless the number of duplicates) is added to JIRA ticket. However if using --dry-run, you can see all the duplicates in the output. I will fix it by removing duplicates.

Reply:
I tried to handle duplicities and also empty strings. Please re-review if you have time. Thanks.